src/safeds/data/tabular/containers/_table.py

```diff
@@ -1057,9 +1057,9 @@ def add_rows(self, rows: list[Row] | Table) -> Table:
         2  5  6
         """
         if isinstance(rows, Table):
-            if rows.number_of_rows == 0:
+            if rows.number_of_columns == 0 or rows.number_of_rows == 0:
                 return self
-            if self.number_of_columns == 0:
+            if self.number_of_columns == 0 or self.number_of_rows == 0:
```

Comment:
We should check after comparing columns to check for different schemas

Reply:
@Marsmaennchen221 But is the schema relevant, if there is nothing changed either way?